mono/eglib/test/path.c

```diff
@@ -299,6 +299,12 @@ test_cwd (void)
 	char *dir = g_get_current_dir ();
 #ifdef G_OS_WIN32
 	const gchar *newdir = "C:\\Windows";
+#elif defined(_AIX)
+	/*
+	 * AIX/PASE have /bin -> /usr/bin, and chdir follows links.
+	 * Use a directory available on both systems that shouldn't be a link.
+	 */
+	const gchar *newdir = "/usr";
```

Comment:
How about `/`?

Reply:
Fair; it'd probably also make sense to collapse it back down to the general POSIX case too.